### ISSUE TITLE
1.1 BASIC STORE

### ISSUE THREAD

**Issue by dawidglap** [reduxjs/redux-devtools#1147]
createStore is actually deprecated
<img width="680" alt="Screenshot 2022-05-03 at 11 17 12" src="https://user-images.githubusercontent.com/81537977/166517370-b58cab8d-6fdd-48ea-85a2-2617a9e4c792.png">

**Comment by jsy135135** [comment]
I think you try  import { legacy_createStore as createStore } from 'redux'

**Comment by jessfeliciano** [comment]
Wouldn't using `createStore` not be recommended since it's no longer supported by redux?

**Comment by jessfeliciano** [comment]
> Wouldn't using `createStore` not be recommended since it's no longer supported by redux?


Wouldn't using createStore not be recommended since it's no longer supported by redux?

**Comment by markerikson** [comment]
Yeah, the _ideal_ answer here is "just use Redux Toolkit's `configureStore` API that does this for you automatically":

```ts
const store = configureStore({
  reducer: {
    counter: counterReducer
  }
})

// automatically added the Redux DevTools configuration
```

The example at https://github.com/reduxjs/redux-devtools/tree/main/extension#installation is still valid code, just not how we'd prefer that people set up and use Redux:

- https://redux.js.org/introduction/why-rtk-is-redux-today
